core/jobs/batch_jobs/skill_validation_jobs.py

```diff
@@ -0,0 +1,85 @@
+# coding: utf-8
```

Comment:
This validation job is no longer needed.

Reply:
Removed.